x/ccv/utils/utils.go

```diff
@@ -51,20 +98,21 @@ func SendIBCPacket(
 	portID string,
 	packetData []byte,
 	timeoutPeriod time.Duration,
-) error {
+) (expiredClient bool, err error) {
+	expiredClient = false
 	channel, ok := channelKeeper.GetChannel(ctx, portID, channelID)
 	if !ok {
-		return sdkerrors.Wrapf(channeltypes.ErrChannelNotFound, "channel not found for channel ID: %s", channelID)
+		return expiredClient, sdkerrors.Wrapf(channeltypes.ErrChannelNotFound, "channel not found for channel ID: %s", channelID)
```

Comment:
might as well return true/false inline. This makes it seem that the client is expired as a quick reader. But really `expiredClient` is false